account-integrations/safe/src/SafeECDSAPlugin.sol

```diff
@@ -60,31 +64,32 @@ contract SafeECDSAPlugin is BaseAccount {
         );
     }
 
-    function enableMyself() public {
+    function enableMyself(address ownerKey) public {
```

Comment:
Since the address storage is now a mapping, it's no longer ideal to set the signing address in the constructor.  So I'm setting the key here instead.  This should also enable other Safe's to use this module as a validator for userops.